receiver/k8sclusterreceiver/internal/collection/collector.go

```diff
@@ -23,9 +23,8 @@ import (
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.uber.org/zap"
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/autoscaling/v2beta1"
+	v2beta2 "k8s.io/api/autoscaling/v2beta2"
```

Comment:
`v2beta2` is redundant alias, this is what k8s.io/api/autoscaling/v2beta2 gives anyway. If we want to provide an alias and avoid conflicts, I'd rename it to `autoscalingv2beta2`

Reply:
sure. I will take care.